serverless/knative_eventing/serverless-kn-trigger.adoc

```diff
@@ -0,0 +1,67 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-kn-trigger"]
+= Using triggers with Knative Eventing
+:context: serverless-kn-trigger
+include::modules/common-attributes.adoc[]
+
+
+toc::[]
+
+All events which are sent to a channel or broker will be sent to all subscribers of that channel or broker by default.
+
+Using triggers allows you to filter events from a channel or broker, so that subscribers will only receive a subset of events based on your defined criteria.
+
+The Knative CLI provides a set of `kn trigger` commands that can be used to create and manage triggers.
+
+.Prerequisites
+Before you can use triggers, you will need:
+
+* Knative Eventing and `kn` installed.
+* An available broker (either the `default` broker, or one that you have created).
+* An available event consumer, for example, a Knative service.
+
+.Example trigger YAML:
+
+----
+apiVersion: eventing.knative.dev/v1alpha1
+kind: Trigger
+metadata:
+  name: trigger-example <1>
+spec:
+ broker: default <2>
+ subscriber:
+    ref:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: my-service <3>
+----
+
+<1> The name of the trigger being created.
+<2> The name of the broker where events will be filtered from. If the broker is not specified, the trigger will revert to using the `default` Broker.
+<3> The name of the service that will consumer filtered events.
+
+[id="create-kn-trigger"]
+== Creating a trigger using `kn`
+
+.Procedure
+
+To create a trigger, enter the following command:
+----
+$ kn trigger create <TRIGGER-NAME> --broker <BROKER-NAME> --filter <KEY=VALUE> --sink <SINK>
```

Comment:
`--filter` shouldn't be mandatory, @dsimansk filed a [bug](https://github.com/knative/client/issues/744) for it yesterday, so it should be updated here as well

Reply:
As Michal stated above. We should probably note that current `kn` requires filter. So no change for now, until it's fixed.